G33 minor change

LVD-AC opened the pull request:
-further named remaining tests
-A and O parameters obsolete
-quick homing with home_delta()
-preventing premature end

LVD-AC committed:
G33 minor change

-further named remaining tests
-A and O parameters obsolete
-quick homing with home_delta()
-preventing premature end

LVD-AC committed:
home_all_axis

bgort commented on Marlin/Marlin_main.cpp:
```diff
@@ -5082,7 +5078,7 @@ void home_all_axes() { gcode_G28(); }
       const int8_t verbose_level = code_seen('V') ? code_value_byte() : 1;
       if (!WITHIN(verbose_level, 0, 2)) {
-        SERIAL_PROTOCOLLNPGM("?(V)erbose level is implausible (0-2).");
+        SERIAL_PROTOCOLLNPGM("?(V)erbose Level is implausible (0-2).");
```
This was changed in a previous merge, and it isn't important really, but it suggests a rebase may be needed?

LVD-AC replied:
I copied the whole G33 routine from a non-Git local copy on my drive I use to develop and beta test. Hence not 100% up to date with the latest merges....

bgort replied:
Yeah, no worries.  Just better to not undo recent changes, if possible.

LVD-AC replied:
done

LVD-AC committed:
L-l-evel

bgort commented:
FWIW, it looks good to me.. but I don't have a delta so can't test.

LVD-AC commented:
tested and all is well

LVD-AC committed:
Merge branch 'bugfix-1.1.x' into G33-minor-update

LVD-AC committed:
"first probe" fail safe

use the first probe result to prevent scraping the bed or floating to
high above it

LVD-AC committed:
1st probe (bis)

LVD-AC commented:
Squashed into #6700
